core/cmd/shell_local.go

```diff
@@ -223,6 +223,29 @@ func initLocalSubCmds(s *Shell, safe bool) []cli.Command {
 						},
 					},
 				},
+				{
+					Name:    "delete-chain",
+					Aliases: []string{},
+					Usage:   "Commands for cleaning up chain specific db tables. WARNING: This will ERASE ALL chain specific data referred to by --chainType and --id options for the specified database, referred to by CL_DATABASE_URL env variable or by the Database.URL field in a secrets TOML config.",
+					Action:  s.CleanupChainTables,
+					Before:  s.validateDB,
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:     "id",
+							Usage:    "chain id based on which chain specific table cleanup will be done",
+							Required: true,
+						},
+						cli.StringFlag{
+							Name:     "chainType",
+							Usage:    "chain type based on which table cleanup will be done, eg. EVM",
+							Required: true,
+						},
+						cli.BoolFlag{
+							Name:  "dangerWillRobinson",
```

Comment:
Can we come up with something a little more standard for this? :thinking: 
Whatever we use, I think the standard for multi-word is to use hyphens:
```suggestion
							Name:  "danger-will-robinson",
```

Reply:
I just copied this from another db shell script so I though it was fine to format it like that. I'll change it to kebab case np

Reply:
Oh, interesting, maybe nbd then :shrug:

Reply:
Yeah, just saw some flags formatted with underscores also, but I think most are with hyphens so I will change it to that